### PR TITLE
feat: add new Cloud Run tag samples

### PR DESCRIPTION
**Pull request by averikitsch**
Samples for https://cloud.google.com/run/docs/rollouts-rollbacks-traffic-migration
Part of b/230161729

If this PR is for Terraform, I acknowledge that I have:

- [x] Searched through the [issue tracker](https://github.com/hashicorp/terraform-provider-google/issues) for an open issue that this either resolves or contributes to, commented on it to claim it, and written "fixes {url}" or "part of {url}" in this PR description. If there were no relevant open issues, I opened one and commented that I would like to work on it (not necessary for very small changes).
- [x] [Generated Terraform](https://github.com/GoogleCloudPlatform/magic-modules#generating-the-terraform-providers), and ran `make test` and `make lint` to ensure it passes unit and linter tests.
- [x] Ensured that all new fields I added that can be set by a user appear in at least one [example](https://github.com/GoogleCloudPlatform/magic-modules/tree/main/templates/terraform/examples) (for generated resources) or [third_party test](https://github.com/GoogleCloudPlatform/magic-modules/tree/main/third_party/terraform/tests) (for handwritten resources or update tests).
- [x] [Ran](https://github.com/hashicorp/terraform-provider-google/blob/main/.github/CONTRIBUTING.md#tests) relevant acceptance tests (If the acceptance tests do not yet pass or you are unable to run them, please let your reviewer know).
- [x] Read the [Release Notes Guide](https://github.com/GoogleCloudPlatform/magic-modules/blob/main/.ci/RELEASE_NOTES_GUIDE.md) before writing my release note below.

**Release Note Template for Downstream PRs (will be copied)**

```release-note:none

```

### CODE REVIEW

File: mmv1/templates/terraform/examples/terraform.yaml_check [hunk 1]
```diff
@@ -0,0 +1,11 @@
+      - !ruby/object:Provider::Terraform::Examples
```
Comment (slevenick) [inline mmv1/templates/terraform/examples/terraform.yaml_check:1]: Should this be removed?

File: mmv1/templates/terraform/examples/cloud_run_service_remove_tag.tf.erb [hunk 2]
```diff
@@ -0,0 +1,18 @@
+resource "google_cloud_run_service" "<%= ctx[:primary_resource_id] %>" {
+  name     = "<%= ctx[:vars]['cloudrun_srv'] %>"
+  location = "us-central1"
+
+  template {}
+
+  traffic {
+    percent       = 100
+    revision_name = "cloudrun-srv-green" # This revision needs to already exist
```
Comment (slevenick) [inline mmv1/templates/terraform/examples/cloud_run_service_remove_tag.tf.erb:9]: If we are trying to actually run these tests these will need to exist and be set up

**Review by slevenick — CHANGES_REQUESTED** [review]
The tests fail probably because the revisions aren't set up. Is there a way to set these up from within Terraform to show a full example?

File: mmv1/templates/terraform/examples/terraform.yaml_check (hunk 1 above)
Comment (averikitsch) [reply mmv1/templates/terraform/examples/terraform.yaml_check]: done

**Comment by slevenick** [comment]
Reassigning as I'm out

File: mmv1/templates/terraform/examples/cloud_run_service_remove_tag.tf.erb (hunk 2 above)
Comment (averikitsch) [reply mmv1/templates/terraform/examples/cloud_run_service_remove_tag.tf.erb]: Ignoring tests similar to other samples

**Comment by averikitsch** [comment]
@pattishin

**Review by c2thorn — APPROVED** [review]

**Pull request merged**